### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by sergiuilie**

### CODE REVIEW

File: com.ca.lsp.cobol/lsp-core-cobol-parser/pom.xml [hunk 1]
```diff
@@ -242,6 +242,11 @@
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
 		</dependency>
+		<dependency>
```
Comment (temanbrcom) [inline com.ca.lsp.cobol/lsp-core-cobol-parser/pom.xml]: Do we need to apply any TPSR process in order to introduce this dependency?

File: com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolPreprocessor.g4 [hunk 2]
```diff
@@ -88,156 +88,270 @@ statement
    ;
 
 moveStatement
-   : MOVE ~(IF | DOT_FS | GO)*?
-   ;
+   : MOVE ~(ACCEPT | ADD | ALTER | CALL | CANCEL | CLOSE | COMPUTE | CONTINUE | DELETE | DISABLE | DISPLAY | DIVIDE | ENABLE |
```
Comment (temanbrcom) [inline com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolPreprocessor.g4]: Is there any way to incapsulate the duplicating code?

**Comment by temanbrcom** [comment]
I confirm that the syntax check at my machine is much faster now. Amazing!

File: com.ca.lsp.cobol/lsp-core-cobol-parser/pom.xml (hunk 1 above)
Comment (sergiuilie) [reply com.ca.lsp.cobol/lsp-core-cobol-parser/pom.xml]: that's a good point, gonna check on it

File: com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolPreprocessor.g4 (hunk 2 above)
Comment (sergiuilie) [reply com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/antlr4/com/ca/lsp/core/cobol/parser/CobolPreprocessor.g4]: it is not duplicate if you are watching closely, every time one keyword is changing but I can try to investigate on that

**Pull request merged**